packages/cadl-ranch-specs/http/models/input-basic/mockapi.ts

```diff
@@ -0,0 +1,19 @@
+import { passOnSuccess, ScenarioMockApi, mockapi, json } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+Scenarios.Basic_Input = passOnSuccess(
+    mockapi.get("/models/basic", (req) => {
+      if (req.body && req.body["requiredString"] && req.body["requiredInt"]) {
+          return {
+            status: 200,
+            body: json({ message: `InputModel was successfully returned` }),
```

Comment:
Where is the cadl model for this `{ message: `InputModel was successfully returned` }`?

Reply:
I will modified as 'InputModel was successfully received'.

Reply:
What I mean is that if you have `Response<200>` in Cadl without a body, you should not return the body in mock api?

Reply:
oh, yes, previously I thought adding body with message can help to validate the response, but validating on status code should be good enough. We should make mockapi.ts consistent with main.cadl file. It is better to remove the the returned body.

Reply:
Yeah. If you want to add the message, modify the Cadl file to return proper model.

Same for the error case, follow the model in core.